pageserver/src/tenant_mgr.rs

```diff
@@ -255,19 +252,27 @@ pub fn get_repository_for_tenant(tenantid: ZTenantId) -> Result<Arc<dyn Reposito
     let m = access_tenants();
     let tenant = m
         .get(&tenantid)
-        .with_context(|| format!("Tenant not found for tenant {}", tenantid))?;
+        .with_context(|| format!("Tenant {} not found", tenantid))?;
 
     Ok(Arc::clone(&tenant.repo))
 }
 
+// Retrieve timeline for tenant without loading whole timeline state into memory
 pub fn get_timeline_for_tenant(
```

Comment:
Interesting, looks like we don't use this function nowhere in the code? 
If we apply YAGNI and remove it, we'll get back to "one method gets and loads the timeline" situation that we've had before, but with a good name now.

Not sure if we really need those trait adjustments now, but not insisting on any changes now either.

Reply:
Removed